server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestGetRepositoriesAction.java

```diff
@@ -51,7 +51,7 @@ public RestGetRepositoriesAction(Settings settings, RestController controller, S
 
     @Override
     public String getName() {
-        return "get_respositories_action";
+        return "get_repositories_action";
```

Comment:
Although this is a valid correction, this is also a breaking change. I think it only affects the REST Usage API, but I think this should be reverted and fixed separately.

Reply:
Reverted and added a FIXME comment there. If the comment is unnecessary, I will remove it.